definitions/variable/sdg-indicators/sdg.yaml

```
@@ -415,3 +470,9 @@
       negative for payment)
     sdg: 17
     unit: billion USD_2010/yr
+- Unemployment:
+    definition: Number of unemployed inhabitants (based on ILO classification)
+    unit: million
+Unemployment [Rate]:
```

Sorry, I guess that this was a mistake in my suggestion…
```suggestion
- Unemployment [Rate]: 
```